### ISSUE TITLE
All bias in conv layers before the batch norm is unnecessary.

### ISSUE THREAD

**Issue by nzw0301** [4uiiurz1/pytorch-nested-unet#39]
When `BatchNorm` with `affine=True` is used, the previous layer's bias is cancelled (See the original BN paper for details). The current implementation's `Conv2D` layers are used with `bias=True`, so by changing `bias=False`, UNet and NestedUnet might be faster.
